test/ecto/repo/autogenerate_test.exs

```diff
@@ -13,18 +13,35 @@ defmodule Ecto.Repo.AutogenerateTest do
     end
   end
 
+  defmodule MeetingRoom do
+    use Ecto.Schema
+
+    schema "meeting_room" do
+      field :name, :string
+      belongs_to :office, Office
+      timestamps type: :utc_datetime_usec
+    end
+
+    def changeset(module, changes) do
+      module
+      |> Ecto.Changeset.cast(changes, [:id, :name])
+    end
+  end
+
   defmodule Office do
     use Ecto.Schema
 
     schema "offices" do
       field :name, :string
-      belongs_to :company, Company
+      belongs_to :company, Ecto.Repo.AutogenerateTest.Company
```

Comment:
The full qualified name is necessary because _I think_ that the module company is not yet defined when compiling `Office` and so the insert call fails saying that the module `Company` is not defined